### ISSUE TITLE
BioModel Discussion Issue for eos9tyg

### ISSUE THREAD

**Issue by Zainab-ik** [ersilia-os/eos9tyg#9]
**Summary !!!**

- **BioModel Name**: Sun2017 -  predictive and interpretable models for PAMPA permeability 7.

- **BioModel Tag**: fair AIML, Machine learning model

- **Metadata Comments**: 
From Original PAMPA publication;

The PAMPA (which is a 7.4) is referred to as a QSPR model (Quantitative Structure-Permeability Relationship) while PAMPA 5.O is referred to as a QSAR model. What's the difference between a QSPR and a QSAR model?. Both are classification models predicting permeability which is a biological activity and that falls under a QSAR category.
I just want to understand if it's a QSAR (note; structure-permeability not structure-property)

- **Contributor**:
> 
 | roles             | Names | 
 | :---------------- | :------: | 
 | Curator        |   @Zainab-ik   | 
 | Code contributor      |   Pauline-Banye       | 
 | Reviewer       |   @GemmaTuron    | 

- **Curation status** :  manually curated

- **Annotation file**: [here](https://docs.google.com/spreadsheets/d/1tRbrVPuBiQpuzGvTmBXDpcoFWtnzBGdkLfJf0l8lcu4/edit?usp=drive_link)

**Comment by GemmaTuron** [comment]
indeed, good question. Let's just use the QSAR notation as it is more standard!

**Comment by GemmaTuron** [comment]
- The publication should be updated: https://slas-discovery.org/article/S2472-5552(22)06765-X/fulltext, can you open a PR
- The output should be good / poor permeability as well to match other models

**Comment by Zainab-ik** [comment]
> * The publication should be updated: https://slas-discovery.org/article/S2472-5552(22)06765-X/fulltext, can you open a PR

Opened here - https://github.com/ersilia-os/eos9tyg/pull/10

> * The output should be good / poor permeability as well to match other models

This is updated.
